### PR TITLE
Week1 Star Wars Assignment

### PR DESCRIPTION
**Pull request by MarieClaireBalabanian**
So the example push is included, it looks like. I am going to research and figure out how to remove commits from repositories via terminal. I would do it before the pull request but a) it's late and you guys probably want the hw and b) i'm scared to mess things up and have to redo everything! But I'm going to figure it out now! 

Also, wanted to know if I was correct in doing the add/commit/push while in the 001_star_wars_cli directory. It makes sense to me...

Many thanks!

### CODE REVIEW

**Comment by billpatrianakos** [comment]
How come you want to remove commits from the repository? Those are there to help you track what your project looked like at a certain point in time. You should get comfortable with the idea that you're not going to get things right the very first time so having multiple commits is common and okay. Even I commit broken code then fix it and commit a second time. It's no big deal.

For now we just want you to get comfortable with the Git workflow (add, commit, push). When we get into advanced Git repo management we can dive into retroactively updating or removing old commits but best practice dictates that you don't mess with a repository history unless you have a really _really_ good reason to.

So long story short, don't worry about fixing old commits. Just fix what you don't like and commit again. 

I wat you to know also that we only get notified of changes to your repo when you hit the "New Pull Request" button. So you can `git commit` a million times before pressing that button and we won't get bombarded with any notifications until you complete the pull request process.

As far as the work goes, I can tell you're comfortable working with the terminal so since there are no letter grades in this class (it's all Pass or Fail) I'd say you passed. Good work!

**Comment by MarieClaireBalabanian** [comment]
Cool. Yeah, I won't worry about those random things! I actually have a quick question. I'm working on my portfolio, and just realized that maybe there are specific guidelines to follow...  Are there? And if so, where? Ive just been doing my thing. Thank you in advance!

**Comment by billpatrianakos** [comment]
This is probably too late of a response but the answer is that there's
really no rules. Just create a single HTML page that has a little "about
me" section and try to make it look as good as you can with the skills and
tools you have on hand. That's all. Simple.

On Thursday, June 2, 2016, MarieClaireBalabanian notifications@github.com
wrote:

> Cool. Yeah, I won't worry about those random things! I actually have a
> quick question. I'm working on my portfolio, and just realized that maybe
> there are specific guidelines to follow... Are there? And if so, where? Ive
> just been doing my thing. Thank you in advance!
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> https://github.com/ga-chicago/wdi5-homework/pull/11#issuecomment-223481108,
> or mute the thread
> https://github.com/notifications/unsubscribe/AA7zwQ6C7DNTePAtZrpFM0P3v1GzHJieks5qH5xpgaJpZM4IsLRT
> .

## 

_Bill Patrianakos_
_Software Engineer_

(815) 412-0921

**Comment by MarieClaireBalabanian** [comment]
Thanks, Bill. Were we supposed to push/pullrequest via github or anything?

**Comment by billpatrianakos** [comment]
For the portfolio you push that to your own "username.github.io"
repository. That one doesn't go in the official homework repo and there's
no pull requesting involved.

I'll try to make it more clear which assignments go in the shared homework
repo and which should just stand on their own from now on.

Sorry to make it so vague.

On Tuesday, June 7, 2016, MarieClaireBalabanian notifications@github.com
wrote:

> Thanks, Bill. Were we supposed to push/pullrequest via github or anything?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> https://github.com/ga-chicago/wdi5-homework/pull/11#issuecomment-224467284,
> or mute the thread
> https://github.com/notifications/unsubscribe/AA7zwZ-c_Dr2zLfgCsXib59Lsdn5SqQzks5qJiKJgaJpZM4IsLRT
> .

## 

_Bill Patrianakos_
_Software Engineer_

(815) 412-0921
